exercises/en/exc_01_03.py

```diff
@@ -1,9 +1,10 @@
-import pandas as pd
+import altair as alt
+from vega_datasets import data
 
 
-candybar_df = pd.read_csv('data/candybars.csv')
+cars = data.cars()
 
-print(candybar_df.head())
-
-candybar_dim = candybar_df.____
-print(candybar_dim)
\ No newline at end of file
+alt.Chart(___).mark___.encode(
+    x=___,
+    y=___,
+    color=___)
```

Comment:
I need to see the data frame to complete the question successfully. I can't remember the column names. Also, I think somewhere we need to explicitly tell people that the column names need to be quoted (for poor R users like me).

Reply:
This is available in the hints section, do you think that is enough?